Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Booking/API.hs

```diff
@@ -193,10 +201,19 @@ buildBookingAPIEntity booking personId = do
   nightIssue <- runInReplica $ QIssue.findNightIssueByBookingId booking.id
   fareBreakups <- runInReplica $ QFareBreakup.findAllByBookingId booking.id
   mbExoPhone <- CQExophone.findByPrimaryPhone booking.primaryExophone
+  (estimatedDistance, estimatedDuration) <- case booking.quoteId of
```

Comment:
booking table has this data, why do we need to fetch it from searchReq? It will add two more queries and impact the perf.